guides/advanced_guides/snapshots_and_dumps.md

```diff
@@ -0,0 +1,101 @@
+
+Dumps, at the difference, are **exporting the data out of MeiliSearch** in a way that they are not bound to the MeiliSearch version. Because of that, when importing a dump, MeiliSearch will need to index all of your documents, a process that takes up time and overhead (based on the amount of documents, their size and the settings).
+
+**To summarize**, **snapshots are highly efficient but not portable between different versions of MeiliSearch. Dumps, on the other hand, are highly portable but not very efficient, as frequently launching MeiliSearch from a dump would cause your performance to suffer.
+
+## Snapshots
+
+A snapshot is an exact copy of the database at the time the snapshot was created. Snapshots do not go through any processing.
+With it, it is possible to create and restore MeiliSearch from snapshots and schedule snapshot creation at custom intervals.
+
+### Creating Snapshots
+
+For MeiliSearch to create snapshots, the feature must be enabled by specifying a path to the directory in which snapshots will be saved.
+
+```bash
+$ meilisearch --snapshot-path mySnapShots/
+```
+
+By default, MeiliSearch schedules a snapshot creation every day. When the instance starts, it will wait 24 hours until the first snapshot. This means that if you have launched MeiliSearch on Tuesday at 4 pm, the first snapshot will be created on Wednesday at 4 pm, and so on every day.
+
+The schedule can be modified with the `--snapshot-interval-sec` flag.
+
+```bash
+$ meilisearch --snapshot-path mySnapShots/ --snapshot-interval-sec 3600
+```
+
+Now a snapshot will be created every hour.
+
+During snapshot creation, snapshots are **overwritten**. Meaning, only the last snapshot will be present in the folder.
+
+[[More about snapshots flags and env variables]](/guides/advanced_guides/configuration.md#schedule-snapshot-creation)
+
+### Start from Snapshot
+
+Because snapshots are zipped copies of your database that did not go through any processing, starting a MeiliSearch instance from one of them is significantly faster.
+
+Using the global environment `MEILI_LOAD_FROM_SNAPSHOT` or the CLI flag `--load-from-snapshot` , MeiliSearch will start the server using the provided snapshot.
+
+```bash
+$ meilisearch --load-from-snapshot mySnapShots/data.ms.tar.gz
+```
+
+If a database already exists, or if no snapshot is found at the given path, MeiliSearch will **stop processing and throw an error**. This is configurable.
+
+If you don't want MeiliSearch to throw an error if a database already exists, you can add the following flag: `--ignore-snapshot-if-db-exists=true`. By adding this flag, if a database exists, MeiliSearch will **not** import the snapshot and continue its process.
```

Comment:
```suggestion
In both cases, **this behavior is [configurable](/guides/advanced_guides/configuration.md#ignore-missing-snapshot)**.

If you don't want MeiliSearch to throw an error when finding that a database already exists, you can add the following flag: `--ignore-snapshot-if-db-exists=true`. When using this flag, MeiliSearch will use the existing database to start an instance instead of throwing an error. The snapshot will be ignored.
```